dev-client/src/screens/AddUserToProjectScreen/components/ProjectRoleRadioBlock.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {useTranslation} from 'react-i18next';
+
+import {ProjectMembershipProjectRoleChoices} from 'terraso-client-shared/graphqlSchema/graphql';
+import {ProjectRole} from 'terraso-client-shared/project/projectSlice';
+
+import {
+  Column,
+  Heading,
+} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {RadioBlock} from 'terraso-mobile-client/components/RadioBlock';
+
+export type Props = {
+  onChange: (value: ProjectMembershipProjectRoleChoices) => void;
+  selectedRole: ProjectMembershipProjectRoleChoices;
+};
+
+export const ProjectRoleRadioBlock = ({onChange, selectedRole}: Props) => {
+  const {t} = useTranslation();
+  return (
+    <Column>
+      <Heading variant="h6">{t('projects.manage_member.project_role')}</Heading>
+      <RadioBlock<ProjectRole>
```

Comment:
issue: the help text is too small to read. @CourtneyLee333 
<img width="318" alt="image" src="https://github.com/techmatters/terraso-mobile-client/assets/86784/fc7455ec-ae15-4b78-900a-381e4a3e9cd5">

Reply:
Hm, this should look like whatever was already being used in the "Manage Team Member" screen. Does it look different to you, or do you think that was already too small?

Reply:
This is what it looked like before my changes in PR #1540  
![Screenshot 2024-06-17 at 3 33 00 PM](https://github.com/techmatters/terraso-mobile-client/assets/5590815/9ac1b7b0-d09c-478d-b2f1-5412c6cbda80)

Reply:
They both seem too small. Your new version has better vertical alignment.

Reply:
This uses the `FormControlHelperText` style in theme.ts, and I suppose raises the question of if it should be changed in all these other places that use it, or in all places that use 12px font size in theme.ts. Some examples:
- ManualSteepnessModal (FormInput helpText)
- IntervalForm (FormControl.HelperText)
- ManageTeamMemberScreen: The caption under "Remove from project" in the screenshot above
- SoilPitSettings label 

I could make this specific location use a different styling than all the other caption or helper styling, but I feel like if we think this is too small, we may want to do a pass later to confirm it makes sense to update it everywhere?

Reply:
Ok, I have added a note about this to https://github.com/techmatters/terraso-mobile-client/issues/1537